tests/scripts/download_gherkin_features.bat

```diff
@@ -0,0 +1,12 @@
+: Adding gherkin feature download for windows
+@echo off
+SET TMP_FOLDER=.\tests\tempFeatures
+SET FOLDER=.\tests\bdd\features
+SET APM_BRANCH=master
+
+mkdir %TMP_FOLDER%
+curl -s https://codeload.github.com/elastic/apm/zip/%APM_BRANCH% -o %TMP_FOLDER%\features.zip
+7z x %TMP_FOLDER%\features.zip -o%TMP_FOLDER% *.feature -y -r
```

Comment:
It's possible that this is totally valid and I just don't know what it means, but the `7z x ` at the start of this line is surprising to me. What does it do?

Reply:
Sure :)

- `7z` stated for `7zip` 
- `x` stated for `extract`

In other words, that's the unzip command, see [7zip](https://www.7-zip.org/)

Reply:
TIL! Thanks!